tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,25 @@
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify service stop operation for Mon and Mgr service
+    """
+    service_name = ["mgr", "mon"]
+
+    # Stop the service
+    node = ceph_cluster.get_nodes(role="installer")[0]
```

Comment:
get test case config parameter `operation` here and perform operation based on the config passed